services/cinema_hall.py

```diff
@@ -0,0 +1,15 @@
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> CinemaHall.objects:
```

Comment:
`CinemaHall.objects` is a manager object. This function does not return it.
Fix it everywhere.